python/dgl/distributed/dist_context.py

```diff
@@ -60,20 +63,25 @@ def initialize(ip_config, num_workers=0, max_queue_size=MAX_QUEUE_SIZE, net_type
     ctx = mp.get_context("spawn")
     global SAMPLER_POOL
     global NUM_SAMPLER_WORKERS
-    if num_workers > 0:
-        SAMPLER_POOL = ctx.Pool(
-            num_workers, initializer=_init_rpc, initargs=(ip_config, max_queue_size,
-                                                          net_type, 'sampler', num_worker_threads))
+    is_standalone = os.environ.get('DGL_DIST_MODE', 'standalone') == 'standalone'
```

Comment:
By default it's standalone mode?

Reply:
yes, it needs to run in Jupyter Notebook by default. The "distributed" mode is turned on by the launch script.